src/libraries/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -533,6 +542,12 @@ public abstract partial class JsonConverter<T> : System.Text.Json.Serialization.
         public abstract T Read(ref System.Text.Json.Utf8JsonReader reader, System.Type typeToConvert, System.Text.Json.JsonSerializerOptions options);
         public abstract void Write(System.Text.Json.Utf8JsonWriter writer, T value, System.Text.Json.JsonSerializerOptions options);
     }
+    [System.AttributeUsageAttribute(System.AttributeTargets.Class | System.AttributeTargets.Struct | System.AttributeTargets.Property | System.AttributeTargets.Field, AllowMultiple = false)]
```

Comment:
Is it meaningful for this to be applied to struct/classes? Does that mean all number types in that class/struct will be treated differently?

Reply:
Yes, that is the intention. All number types in the class/struct will be treated in line with the handling specified.